src/models/DeviceModel.js

```diff
@@ -0,0 +1,18 @@
+import { observable, action, computed } from "mobx";
+
+
+export default class DeviceModel {
```

Comment:
There's no need to have Model in class name.
But it can stay like it is right now